spec/system/planning_applications/reviewing_spec.rb

```diff
@@ -30,6 +30,8 @@
       expect(page).to have_content("New assessor comment")
     end
 
+    expect(page).to have_no_link("Validate documents")
```

Comment:
I think the task_list_spec.rb covers this, so I don't think we need to put this check in here as well. (keen to keep these feature specs small and manageable)